controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -1835,26 +1831,26 @@ private SegmentRange convert(String scope, String stream, Map.Entry<Long, Map.En
                 segment.getValue().getValue());
     }
 
-    private UpdateStreamStatus.Status handleUpdateStreamError(Throwable ex, long requestId) {
+    private UpdateStreamStatus.Status handleUpdateStreamError(Throwable ex, long requestId, String logMessage, String streamFullName) {
         Throwable cause = Exceptions.unwrap(ex);
+        log.error(requestId, "Exception updating Stream {}. Cause: {}, {} ", streamFullName, logMessage, cause);
```

Comment:
Is the third {} required ? would the following suffice.
```
log.error(requestId, "Exception updating Stream {}. Cause: {} ", streamFullName, logMessage, cause);
```

Reply:
Yes. There are 3 arguments, so 3 pairs of braces are needed, log-message is different from cause.

Reply:
Yes you were right. I have changed it to 2 braces now.